rest-jersey/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1 @@
+jersey.config.server.application.name=load properties test
```

Comment:
@derekm Please move this file to the test resources

Reply:
Yes, I will! Actually, that's where I thought it was! Good eye!

Reply:
Yeah, I missed that too.  good catch.